### PR TITLE
Configure: add configure command line C flags after the configured C flags

### LINKED ISSUE #5203: enable-ubsan failure
run-checker is reporting a failure with these config arguments:

````
$ CC=clang ../openssl/config -d --strict-warnings enable-ubsan -DPEDANTIC -DOPENSSL_SMALL_FOOTPRINT -fno-sanitize=alignment
````

It seems to get stuck in test_comp.

````
$ V=1 make TESTS=test_comp test
make depend && make _tests
make[1]: Entering directory '/home/matt/dev/openssl-write'
make[1]: Leaving directory '/home/matt/dev/openssl-write'
make[1]: Entering directory '/home/matt/dev/openssl-write'
( cd test; \
  mkdir -p test-runs; \
  SRCTOP=../. \
  BLDTOP=../. \
  RESULT_D=test-runs \
  PERL="/usr/bin/perl" \
  EXE_EXT= \
  OPENSSL_ENGINES=`cd .././engines; pwd` \
  OPENSSL_DEBUG_MEMORY=on \
    /usr/bin/perl .././test/run_tests.pl test_comp )
../test/recipes/70-test_comp.t .. 
Proxy started on port 4453
Server command: ../../util/shlib_wrap.sh ../../apps/openssl s_server -no_comp -rev -engine ossltest -accept 4443 -cert ../../apps/server.pem -cert2 ../../apps/server.pem -naccept 1 -cipher AES128-SHA:TLS13-AES-128-GCM-SHA256
Client command: echo test | ../../util/shlib_wrap.sh ../../apps/openssl s_client -engine ossltest -connect localhost:4453
crypto/cryptlib.c:82:13: runtime error: store to misaligned address 0x7fb4210701dc for type 'unsigned int', which requires 8 byte alignment
crypto/cryptlib.c:82:13: runtime error: store to misaligned address 0x7fb58c7011dc for type 'unsigned int', which requires 8 byte alignment
0x7fb58c7011dc: note: pointer points here
0x7fb4210701dc: note: pointer points here
  00 00 00 00 ab 27 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00
              ^ 
  00 00 00 00 ab 27 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00
              ^ 
````

I don't understand the error here:

````
runtime error: store to misaligned address 0x7fb58c7011dc for type 'unsigned int', which requires 8 byte alignment
````

This was run on my Linux 64-bit machine. The error message claims that an `unsigned int` requires 8 byte alignment. but according to this document (https://software.intel.com/sites/default/files/article/402129/mpx-linux64-abi.pdf) the alignment required for an `unsigned int` is 4 bytes.

Making this change does make the problem go away:

````
diff --git a/crypto/x86_64cpuid.pl b/crypto/x86_64cpuid.pl
index 0a88c7a4ed..eff924283d 100644
--- a/crypto/x86_64cpuid.pl
+++ b/crypto/x86_64cpuid.pl
@@ -31,7 +31,7 @@ print<<___;
        call    OPENSSL_cpuid_setup
 
 .hidden        OPENSSL_ia32cap_P
-.comm  OPENSSL_ia32cap_P,16,4
+.comm  OPENSSL_ia32cap_P,16,8
 
 .text
````
But this doesn't seem like a very satisfactory answer. Is this a clang/ubsan bug? Or have I misunderstood something?

### PR DESCRIPTION
**Pull request by levitte**
Fixes #5203

### CODE REVIEW

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
Hmm, Travis is unhappy

**Comment by levitte** [comment]
Seems that was a Travis hickup.  Trying a re-run

File: Configure [hunk 1]
```diff
@@ -1492,6 +1484,22 @@ unless ($disabled{afalgeng}) {
...
+    my $target_key = $user_to_target{$_} // lc $_;
+
+    if (defined $config{$target_key}) {
+        push @{$config{$target_key}}, @{$useradd{$_}};
+    } else {
+        $config{$target_key} = [ @{$useradd{$_}} ];
+    }
+}
+
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
```
[the remark this answers is not in the document]
Comment (levitte) [reply Configure]: I don't see why.  It's fine if someone wants to use %useradd later on...  just not modify it

**Comment by levitte** [comment]
Merged. 63468812bdbee64bb3ca757c4692cb3cf2ace01e and 26f0340d49138d489ffc9662dff827488a12142c

... and I didn't notice the appveyor failure before merging, sorry...  A corrective PR coming up,

File: Configure (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmshort) [reply Configure]: Mostly just to mirror the deleted comment from earlier.

[the remark this answers is not in the document]
Comment (levitte) [reply Configure]: Yeah well, this is so late in the configuration process that I figured the chances are much smaller

[the remark this answers is not in the document]
Comment (tmshort) [reply Configure]: No worries then

**Pull request closed without merging**